cmd/deployment.go

```diff
@@ -409,8 +414,16 @@ func deploymentCreate(cmd *cobra.Command, args []string, client *houston.Client,
 func deploymentDelete(cmd *cobra.Command, args []string, client *houston.Client, out io.Writer) error {
 	// Silence Usage as we have now validated command input
 	cmd.SilenceUsage = true
+	if hardDelete {
+		i, _ := input.InputConfirm(
+			fmt.Sprintf(messages.CLI_DEPLOYMENT_HARD_DELETE_PROMPT))
 
-	return deployment.Delete(args[0], client, out)
+		if !i {
+			fmt.Println("Not permanently deleting...")
```

Comment:
This is confusing here. We shouldn't tell the user the negative when it's the default behavior. So just `Deleting ...` should be sufficient.

Reply:
should I not put a message? I just wanted to put something indicating it wasn't going to permanently remove it if they say no

Reply:
@paolaperaza thoughts?

Reply:
If they say "no" to the prompt, it should exit and not execute any command.

Reply:
You can even have a message "No command was executed" so that the user knows. But we can't assume the user wants to soft delete a deployment if they say no to a hard delete.

Reply:
oh, i thought if u say no u continue the operation just not `hard`, so I should exit?

Reply:
will refactor

Reply:
I think that if the user specifies the option to hard delete and then when asked to confirm if they are sure and they say no, we should exit with a message saying something like the "command was not executed" and have them do the soft delete again if that is what they want to do.
But @paolaperaza or @jwitz can wordsmith this better.

Reply:
👋   For this message, maybe:

> Exit: This command was not executed and your Deployment was not hard deleted.

If we wanted to give more information/a next step, we could say:

> Exit: This command was not executed and your Deployment was not hard deleted. If you want to delete your Deployment but not permanently,  try `$ astro deployment delete` without the `--hard` flag.

Reply:
added the give more information text, updated pr description with how it looks